var/spack/repos/builtin/packages/enzo/package.py

```diff
@@ -53,16 +62,29 @@ def edit(self, spec, prefix):
             filter_file('^LOCAL_HYPRE_INSTALL.*',
                         'LOCAL_HYPRE_INSTALL =',
                         'Make.mach.spack')
+            if spec.satisfies('%gcc@10.2.0:'):
+                filter_file(r'^MACH_FFLAGS\s*=\s*',
+                            'MACH_FFLAGS   = -fallow-argument-mismatch ',
+                            'Make.mach.spack')
+                filter_file(r'^MACH_F90FLAGS\s*=\s*',
+                            'MACH_F90FLAGS = -fallow-argument-mismatch ',
+                            'Make.mach.spack')
 
     def build(self, spec, prefix):
         with working_dir('src/enzo'):
             make('machine-spack')
-            make('opt-high')
+            make('opt-' + self.spec.variants['opt'].value)
             make('show-config')
             make()
+        with working_dir('src/inits'):
+            make()
+        with working_dir('src/ring'):
+            make()
 
     def install(self, spec, prefix):
         install_tree('bin', prefix.bin)
         install_tree('doc', prefix.doc)
         install_tree('input', prefix.input)
         install_tree('run', prefix.run)
+        with working_dir(join_path('src', 'ring')):
+            install('ring.exe', join_path(prefix.bin, 'ring'))
```

Comment:
Why not simply do the following?
```python
install('src/ring/ring.exe', join_path(prefix.bin, 'ring'))
```

Reply:
Huh, I think it should work. But I don't have time to test it right now, can we skip on it?

Reply:
I also built Enzo with this change and it works.